ajax/libs/vanilla-lazyload/package.json

```
@@ -1,24 +1,34 @@
 {
   "name": "vanilla-lazyload",
   "filename": "lazyload.min.js",
-  "version": "12.3.0",
-  "description": "LazyLoad is a fast, lightweight and flexible script for loading images only when they're about to enter the viewport of a scrollable area, with an excellent support to the progressive JPEG image format. Typescript module definition is available.",
+  "version": "15.1.1",
```

```suggestion
  "version": "12.3.0",
```
Version should not be changed, the auto-update bot will do this.